examples/simple/main.tf

```diff
@@ -49,6 +48,30 @@ module "materialize_infrastructure" {
   enable_monitoring      = true
   metrics_retention_days = 3
 
+  # Enable and configure Materialize operator
+  install_materialize_operator = true
+
+  # Once the operator is installed, you can define your Materialize instances here.
+  # Uncomment the following block (or provide your own instances) to configure them.
+  # materialize_instances = [
```

Comment:
Just wondering ... we have instructions that has people uncommenting these ... but, could `materialize_instances` be a variable? So, that, subsequent runs include a tfvars file?

Reply:
Yep, totally! Just updated the example. Also added an example tfvars in the examples directory as a reference!

Reply:
Thank you!